content/300-guides/200-deployment/110-deployment-guides/450-deploying-to-cloudflare-workers.mdx

```diff
@@ -150,7 +150,7 @@ Now you're ready to add Prisma to the project.
 npx prisma init
 ```
 
-This will create a Prisma Schema in `prisma/schema.prisma` and an `.env` file.
+This will create a Prisma Schema in `prisma/schema.prisma`.
```

Comment:
This does actually create an `.env` file. Are you removing the mention because it is not always useful for Data Proxy Client @millsp?

Reply:
Yes, it does create one, removing as it is not useful here. Should we add an admonition here too?

Reply:
Having liaised with Pierre, I'm going to add a note saying that the file does get created, but that you cannot use it in conjunction with Cloudflare Workers.

Reply:
And while I'm at it, I'll improve the wording slightly.